tools/webots.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+
+from termcolor import cprint
+
+import b
+
+
+def register(command):
+
+    command.help = "Build and run images for use with Webots simulation environment"
+    subparsers = command.add_subparsers(help="sub-command help", dest="sub_command")
+
+    command_build = subparsers.add_parser("build")
+    command_run = subparsers.add_parser("run")
+
+    command_build.add_argument(
+        "-c",
+        "--clean",
+        dest="clean",
+        action="store_true",
+        default=False,
+        help="Cleans config (./b configure --clean)",
+    )
+
+    command_build.add_argument(
+        "-p",
+        "--push",
+        dest="push",
+        action="store_true",
+        default=False,
+        help="Pushes image to Docker Hub",
+    )
+
+    command_run.add_argument("role", nargs=1, help="Role to run")
+
+
+def run(sub_command, role=None, **kwargs):
+    if sub_command == "build":
+
+        # # Set target
+        # err = subprocess.run(["./b", "target", "robocup2021"]).returncode
+        # if err != 0:
+        #     cprint("returned exit code {}".format(err), "red", attrs=["bold"])
+        #     exit(err)
+
+        # Remove build volume
+        if kwargs["clean"] == True:
+            # Clean config
+            err = subprocess.run(["./b", "configure", "--clean"]).returncode
+            if err != 0:
+                cprint("returned exit code {}".format(err), "red", attrs=["bold"])
+                exit(err)
+
+        # Set build config
+        err = subprocess.run(
+            ["./b", "configure", "--", "-DCMAKE_BUILD_TYPE=Release", "--", "-DROLE_webots=ON"]
```

Comment:
double `"--"`? I don't follow what the second `"--"` is meant to do

Reply:
Woops, the second `"--"` shouldnt be there